sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -187,321 +222,263 @@ client_product_feature_usage AS (
     --awesomebar_bottom
-    COUNT(DISTINCT CASE WHEN awesomebar_bottom > 0 THEN client_id END) AS awesomebar_bottom_users,
-    SUM(awesomebar_bottom) AS awesomebar_bottom,
+  COUNT(DISTINCT CASE WHEN awesomebar_bottom > 0 THEN client_id END) AS awesomebar_bottom_users,
     --awesomebar_null
-    COUNT(DISTINCT CASE WHEN awesomebar_null > 0 THEN client_id END) AS awesomebar_null_users,
-    SUM(awesomebar_null) AS awesomebar_null,
+  COUNT(DISTINCT CASE WHEN awesomebar_null > 0 THEN client_id END) AS awesomebar_null_users,
     /*Notificaion*/
     --metrics_notifications_allowed
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN metrics_notifications_allowed > 0
-          THEN client_id
-      END
-    ) AS metrics_notifications_allowed_users,
-    SUM(metrics_notifications_allowed) AS metrics_notifications_allowed,
+  COUNT(
+    DISTINCT
+    CASE
+      WHEN metrics_notifications_allowed > 0
+        THEN client_id
+    END
+  ) AS metrics_notifications_allowed_users,
+  SUM(metrics_notifications_allowed) AS metrics_notifications_allowed,
     --events_marketing_notification_allowed
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN events_marketing_notification_allowed > 0
-          THEN client_id
-      END
-    ) AS events_marketing_notification_allowed_users,
-    SUM(events_marketing_notification_allowed) AS events_marketing_notification_allowed,
+  COUNT(
+    DISTINCT
+    CASE
+      WHEN events_marketing_notification_allowed > 0
+        THEN client_id
+    END
+  ) AS events_marketing_notification_allowed_users,
+  SUM(events_marketing_notification_allowed) AS events_marketing_notification_allowed,
     /*Customize Homepage*/
     --customize_home_contile
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN customize_home_contile > 0
-          THEN client_id
-      END
-    ) AS customize_home_contile_users,
-    SUM(customize_home_contile) AS customize_home_contile,
+  COUNT(
+    DISTINCT
+    CASE
+      WHEN customize_home_contile > 0
+        THEN client_id
+    END
+  ) AS customize_home_contile_users,
+  SUM(customize_home_contile) AS customize_home_contile,
     --customize_home_jump_back_in_users
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN customize_home_jump_back_in > 0
-          THEN client_id
-      END
-    ) AS customize_home_jump_back_in_users,
-    SUM(customize_home_jump_back_in) AS customize_home_jump_back_in,
+  COUNT(
+    DISTINCT
+    CASE
+      WHEN customize_home_jump_back_in > 0
+        THEN client_id
+    END
+  ) AS customize_home_jump_back_in_users,
+  SUM(customize_home_jump_back_in) AS customize_home_jump_back_in,
     --customize_home_most_visited_sites_users
-    COUNT(
-      DISTINCT
-      CASE
-        WHEN customize_home_most_visited_sites > 0
-          THEN client_id
-      END
-    ) AS customize_home_most_visited_sites_users,
-    SUM(customize_home_most_visited_sites) AS customize_home_most_visited_sites,
+  COUNT(
```

Comment:
Could we update the `COUNT DISTINCT CASE` to be `COUNT DISTINCT IF` to be consistent with the ios query:
https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/query.sql#L254